code/modules/mob/living/silicon/ai/life.dm

```diff
@@ -160,14 +160,19 @@
 		return
 
 	if(ai_flags & COREFORTIFY)
-		brute_damage_modifier = 0.25
+		brute_damage_modifier = 0.33
```

Comment:
since this value is different than the old value, please note it in your document